packages/frontend/package.json

```diff
@@ -32,6 +33,7 @@
     "near-api-js": "^0.43.1",
     "near-ledger-js": "^0.2.1",
     "near-seed-phrase": "^0.2.0",
+    "node-cache": "^5.1.2",
```

Comment:
i believe that the repo is using redux as a cache of sorts, ie functions that reference "getting from the cache" are just pulling from the redux store. node-cache is needed to specify a TTL on the dataloader cache, because dataloader out of the box is only a short lived per request cache. we want something with a little longer TTL (30 seconds). this is also a dependency pulled from near-contract-helper.